docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -101,43 +101,50 @@ specific event in the sequence, update the rule's EQL statement. For example:
 --
 * To add an exception from the rule details page:
 .. Go to the rule details page of the rule to which you want to add an
-exception (*Detect* -> *Rules* -> *_<Rule name>_*).
+exception (*Manage* -> *Rules* -> *_<Rule name>_*).
 .. Scroll down below the rule details and select the *Exceptions* tab.
 +
 [role="screenshot"]
 image::images/exception-histogram.png[Detail of Exceptions tab, 75%]
 .. Click *Add new exception* -> *Add rule exception*.
 
 * To add an exception from the Alerts table:
-.. Go to *Detect* -> *Alerts*.
+.. Go to *Alerts*.
 .. Scroll down to the Alerts table, go to the alert you want to create an exception for, click the *More Actions* menu (*...*), then select *Add rule exception*.
 
 The *Add Rule Exception* flyout opens (the example below was opened from the Alerts table):
 --
 +
 [role="screenshot"]
 image::images/add-exception-ui.png[]
-. Add conditions that define when the exception prevents alerts. You can define
-multiple conditions with `OR` and `AND` relationships. In the example above,
-the exception prevents the rule from generating alerts when the
+. Use the following settings to add conditions that define when the exception prevents alerts. In the example above, the exception prevents the rule from generating alerts when the
 `svchost.exe` process runs on agent hostname `siem-kibana`.
-+
-[IMPORTANT]
-============
-* You can use nested conditions. However, this is only required for
-<<nested-field-list, these fields>>. For all other fields, nested conditions
-should not be used.
 
-* Wildcards are not supported in rule exceptions or value lists. Values must be literal values.
-============
+  .. *Field*: Select a field to identify the event being filtered.
+
+  .. *Operator*: Select an operator to define the condition:
+    * `is` | `is not` — Must be an exact match of the defined value.
+    * `is one of` | `is not one of` — Matches any of the defined values.
+    * `exists` | `does not exist` — The defined field exists.
+    * `is in list` | `is not in list` — Matches values in a value list.
```

Comment:
```suggestion
    * `is in list` | `is not in list` — Matches values in a list.
```

Reply:
I think this needs to remain "value list" to be consistent with other usage in the topic and also in the UI (the button on the Rules page is "Import value lists".)